pip-dep/requirements.txt

```diff
@@ -3,13 +3,15 @@ flask_socketio>=3.3.2
 lz4>=2.1.6
 msgpack>=0.6.1
 numpy>=1.16.0
-phe>=1.4.0
-Pillow<7
 scipy>=1.4.1
 syft-proto>=0.1.1.a1.post2
 tblib>=1.4.0
 torch==1.4
 torchvision==0.5.0
+phe>=1.4.0
+Pillow<7
+requests==2.22.0
+scikit-learn==0.20.4
```

Comment:
I think this could be removed since we also have another requirement file needed for the notebooks :)